doc/api/util.md

```diff
@@ -239,6 +239,23 @@ The `--throw-deprecation` command-line flag and `process.throwDeprecation`
 property take precedence over `--trace-deprecation` and
 `process.traceDeprecation`.
 
+## `util.deprecate.isDeprecated`
```

Comment:
It would seem more user-friendly to have `util.isDeprecated(value)` which does the check using the symbol internally